git/url_test.go

```diff
@@ -196,13 +196,22 @@ func TestParseURL(t *testing.T) {
 				Path:   "",
 			},
 		},
+		{
+			name:    "fails to parse",
+			url:     "ssh://git@[/tmp/git-repo",
+			wantErr: true,
+		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			u, err := ParseURL(tt.url)
-			if (err != nil) != tt.wantErr {
```

Comment:
This was just buggy since a failure to parse returns a `nil` url which would panic on the subsequent assertions. No test was actually using this branch.